vm/systemSmartContracts/delegation.go

```diff
@@ -1140,6 +1206,13 @@ func (d *delegation) unStakeNodes(args *vmcommon.ContractCallInput) vmcommon.Ret
 		return vmcommon.UserError
 	}
 
+	entry := &vmcommon.LogEntry{
```

Comment:
I do not see why should we add this entry - this does not have any value. The same for everywhere you add directly hex decoded data as logEntry.

Reply:
removed.

Reply:
done.